components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -204,6 +204,10 @@
     private ValueGenerator tokenValueGenerator;
 
     private String tokenValueGeneratorClassName;
+
+    // Property added to determine the expiration of logout token in oidc back-channel logout.
+    private String openIDConnectBCLogoutTokenExpiryInSeconds = "120";
```

Comment:
can we make this final? so no body can change this.

Reply:
We are changing this expiry value based on the configuration in identity.xml file in a method in this class. So we can't make this as final